src/com/lftechnology/java_training/jan7/RenameAndDeleteOrgFile.java

```diff
@@ -0,0 +1,116 @@
+package com.lftechnology.java_training.jan7;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.logging.Level;
+
+/**
+ * Write a program to rename the given file, after renaming the file delete the renamed file. (Accept the file name using command line
+ * arguments.)
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class RenameAndDeleteOrgFile {
+	private static Scanner input = new Scanner(System.in);
+	private static TrainingConstantsUtility constantValues = new TrainingConstantsUtility();
+
+	public static void main(String[] args) {
+		String path = "secondprogramfile.txt";
+		File newFile = new File(path);
+		createFile(newFile);
+		newFile = renameFile(newFile);
+		deleteFile(newFile);
+	}
+
+	/**
+	 * creates new file and checks if it already exists or not
+	 * 
+	 * @param newFile
+	 */
+	private static void createFile(File newFile) {
+		// check if the file already exist or not
+		try {
+			if (!newFile.exists()) {
+				newFile.createNewFile();
+				constantValues.LOGGER.info("New file created");
+			} else {
+				constantValues.LOGGER.info("file already exists");
+			}
+
+		} catch (IOException e) {
+			constantValues.LOGGER.log(Level.INFO, "error occured {0}", e);
+
+		}
+	}
+
+	/**
+	 * ask user to enter new name to rename the old filename
+	 * 
+	 * @param oldFile
+	 *            an object of {@link File}
+	 * @return fil with renamed filename
```

Comment:
File spelling. And use class in java doc so that the user can reference the Class file. It'll look like {@link File}